README.md

```diff
@@ -39,30 +39,115 @@ app.post('/api/cart/:userId', (req, res) => {
   // wait to revieve the event of the click to know what button what pushed and where to post the info.
 });
 
-Responses:
+## Expected Response
+
 Add to cart:
+
 Success(200)
+
 Console.log(Data was succesfully added to Cart)
 
 ## Data Schema for Cart
 {
-  id:
-  user_id:
-  product_id:
-  subtotal:
+  id: Number
+  user_id: Number
+  product_id: Number
+  subtotal: Number
+}
+
+
+## Expected Response
+
+Add to list:
+
+Success(200)
+
+console.log(Data was succesfully added to List)
+
+## Data Schema for List
+{
+  id: Number
+  list_type: Number
+  product_id: Number
+  user_id: Number
 }
 
+## PUT Request
+
+app.put('/api/cart/:productId', (req, res) => {
```

Comment:
This example should be of the call a user would use to send data to the API endpoint. E.g. an AJAX call example.